constants.py

```diff
@@ -24,3 +25,8 @@
                             border-width: 2px;
                             border-color: blue;
                         }"""
+
+
+class AddStreamError(Enum):
+    url_not_supported = 1
```

Comment:
`auto()` apparently is a python 3.6 feature, I'll leave it as 1 and 2 to support more python versions 👍